projects/widget-test-app/.env.sample

```diff
@@ -4,3 +4,8 @@ VITE_IMPERSONATE_USER_ID=
 VITE_PAYMENT_SESSION_URL_OVERRIDE=
 VITE_ENTERPRISE_CORE_API_BASE_URL=https://api.enterprise.sandbox.uphold.com/core
 VITE_ENTERPRISE_WIDGETS_API_BASE_URL=https://api.enterprise.sandbox.uphold.com/widgets
+VITE_PAYMENT_WIDGET_FLOW=select-for-withdrawal
+VITE_CREATE_QUOTE_ORIGIN_ID=
```

Comment:
I don't like how this is growing.

Question: do we really need a test app, now that we have the `/test` facility? If so, I would prefer to improve the test app so that once I click to test the payment widget:

- It will ask me which flow I want to run.
- For the authorize flow, it will allow pasting a `quote request body`, which is already pre-populated.

Reply:
I refactored the page and the configuration system to make it easier to implement new flows in the future. In the case of the `authorize` flow, the `quote request body` will be set in a json file which will be read by the test app.